sematic/BUILD

```diff
@@ -11,6 +11,8 @@ sematic_py_lib(
         "//sematic:future_context",
         "//sematic/future_operators:init",
         "//sematic/plugins/external_resource:timed_message",
+        "//sematic/plugins/kuberay_wrapper:standard",
+        "//sematic/plugins/external_resource/ray:cluster",  # TODO: don't merge!!!
```

Comment:
Just temporary; will remove once we get a way to ship a separate server image figured out